src/pages/settings/project/ProjectSettings.jsx

```diff
@@ -61,6 +61,30 @@ const ProjectSettings = () => {
       .finally(() => dispatch(decrement("pendingRequests")));
   };
 
+  const handleAES = aesKey => {
+    dispatch(increment("pendingRequests"));
+    client.projects
+      .setProjectGlobalConfig(projectID, Object.assign(globalConfig, {}, { aesKey: aesKey }))
+      .then(() => {
+        setProjectConfig(projectID, "aesKey", aesKey);
+        notify("success", "Success", "Changed AES Key successfully");
+      })
+      .catch(ex => notify("error", "Error", ex))
+      .finally(() => dispatch(decrement("pendingRequests")));
+  };
+
+  const handleContextTimeout = contextTimeout => {
+    dispatch(increment("pendingRequests"));
+    client.projects
+      .setProjectGlobalConfig(projectID, Object.assign(globalConfig, {}, { contextTimeout: contextTimeout }))
```

Comment:
This is wrong. In object.assign, we pass the empty object as the first parameter usually.